### PR TITLE
add the img path to the function process_image

### PR DESCRIPTION
**Pull request by BeyondYourself**
The purpose of adding image path is to distinguish which image has the problem when we process the images

### CODE REVIEW

**Comment by BeyondYourself** [comment]
@tink2123 please review this pr,thx

**Comment by BeyondYourself** [comment]
@littletomatodonkey  please review this pr ,thx

File: ppocr/data/rec/img_tools.py [hunk 1]
```diff
@@ -339,6 +339,7 @@ def warp(img, ang):
 
 
 def process_image(img,
+                  img_path,
```
Comment (littletomatodonkey) [inline ppocr/data/rec/img_tools.py:342]: 把添加的img path参数写在最后把，同时指定`img_path=""`，防止影响其他的函数，上面的参数顺序也调整下

Comment (BeyondYourself) [reply ppocr/data/rec/img_tools.py]: OK

**Pull request closed without merging**